experiments/model_c/api.py

```diff
@@ -0,0 +1,72 @@
+"""
+tts package supports using command line or python to call model for training
+or inferencing. Here we deomnstrate how to use python to call models
+"""
+from typing import Optional, Union
+import sounddevice as sd
+from scipy.io import wavfile
+import torch
+from TTS.api import TTS
+
+
+DeviceLikeType = Union[str, torch.device, int]
```

Comment:
From 3.11, you could write it this way:
```python
from typing import TypeAlias

DeviceLikeType: TypeAlias = str | torch.device | int
```

Reference: https://peps.python.org/pep-0604/

Reply:
updated, thanks!